docs/saml2aws-auto.1

```diff
@@ -0,0 +1,105 @@
+.TH saml2aws-auto 1 "01 Oct 2018" "version 1.0"
+
+.SH NAME
+saml2aws-auto - A simple to use management tool for multiple AWS account credentials when using Keycloak as Identity Provider
+.SH SYNOPSIS
+saml2aws-auto [-hvV] <SUBCOMMAND>
+.SH DESCRIPTION
+The following
+.B flags
+are available:
+
+.B  -h, --help
+    Display a help text.
+
+.B  -V, --version
+    Display version information.
+
+.B  -v
+    Enable more verbose output (Denoted with 'DEBU' in output).
+
+
+The following
+.B subcommands
+are available:
+
+.B  configure
+    Interactively configure Identity Provider endpoint and user login credentials.
+
+.B  refresh
+    Renew downloaded STS credentials by reauthenticating with IDP.
+
+.B  groups
+    Add a logical group of AWS roles for federation.
+
+    The following
+.B subcommands
+are available:
+
+        groups
+.B add
+<name>
+          Adds a group with one or more roles as <name>.
+          This subcommand of
+.B groups
+has the following options:
+
+          --accounts <accounts>
```

Comment:
can you add a note that either `accounts` or `prefix` has to be supplied, but not both?

Reply:
Done